rts/lra/lra-coordinator-jar/src/main/resources/META-INF/publicKey.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAlivFI8qB4D0y2jy0CfEq
```

Comment:
Again as mentioned in my other comment pack the key under the `src/main/resources` which means this key goes to the LRA release jar is a bad practice. This is used for testing so it has to be placed under `src/test/resources` and linked in tests dynamically. We can't release the runner jar with some private/public keys inside.

Reply:
I will remove this.